src/Tools/AI Test Toolkit/src/TestSuite/AITTestSuite.Page.al

```diff
@@ -156,6 +156,30 @@ page 149031 "AIT Test Suite"
                     CurrPage.Update(false);
                 end;
             }
+            action(StartBatch)
+            {
+                Enabled = Rec.Status <> Rec.Status::Running;
+                Caption = 'Start Batch';
+                Image = ExecuteBatch;
+                ToolTip = 'Starts running the AI Test Suite, the specified number of times.';
+
+                trigger OnAction()
+                var
+                    AITBatchRunDialog: Page "AIT Batch Run Dialog";
+                    Iterations: Integer;
+                begin
+                    CurrPage.Update(false);
```

Comment:
Why the need to update?

Reply:
The normal start action updates the page before and after executing suite, this action should mirror that action.

The last one is to update the results after the run.

This one is probably not so necessary but I'm thinking this update is to update the results on the page before running, so it shows most recent results until this execution finishes (which can take a while). Execution is in a separate thread, so results could have been updated since accessing the page.

Reply:
I guess i can understand, but could user not just hit f5 ? If they wanted? 
Once something completes, the update on 180 would ensure to update it, if i understand correctly.